ava.config.js

```diff
@@ -1,5 +1,4 @@
 export default {
-  compileEnhancements: false,
```

Comment:
Why are you removing this?

Reply:
Because ava: 

https://github.com/avajs/ava/issues/2376#issuecomment-576364924

> Oh that setting is no longer needed. It should be removed from the documentation. I see it in two different places in that file, perhaps it's mentioned elsewhere too.

Ava's update was a breaking change and that configuration is no longer needed for typescript

Reply:
Thanks